### DISCUSSION TITLE
What is Upscale ?

### DISCUSSION THREAD

**Discussion by oldhand7** [lllyasviel/Fooocus#2768] [Q&A]
I 've seen there are upscaling parameters . I would liket to know what is upscale, denoising and how these values affect the image generation.

**Comment by Rogala** [comment]
[New Edition. Handbook of working with Fooocus 2023-2024.pdf](https://github.com/lllyasviel/Fooocus/files/15006706/New.Edition.Handbook.of.working.with.Fooocus.2023-2024.pdf)
